app/experimenter/kinto/README.md

```diff
@@ -0,0 +1,158 @@
+- Live: An experiment in live can not be edited and is published to the collection corresponding to its target application after it has been reviewed in Experimenter and in Remote Settings
+- Complete: An experiment in complete can not be edited and is no longer published in Remote Settings
+
+#### Publish States
+
+- Idle: An experiment has no changes that require review or modification in Remote Settings
+- Review: An experiment has changes that require review in Experimenter before they can be published to Remote Settings
+- Approved: An experiment has changes that have been approved in Experimenter and must be published to Remote Settings
+- Waiting: An experiment has changes that have been published to Remote Settings and are awaiting further review in Remote Settings
+
+In theory an experiment can occupy any combination of these two states, but in practice it will only have a publish state other than Idle in Draft and in Live. Preview experiments can be modified in Remote Settings without any review, and Complete experiments will never be published to Remote Settings.
+
+## Workflows
+
+The following diagrams describe every interaction between Experimenter and Remote Settings. An experiment's state is denoted as `(lifecycle state, publish state)`. Any change which requires creating a changelog includes `+changelog`.
+
+The following actors are involved:
+
+- Experiment Owner: The user in Experimenter that creates and edits an experiment
+- Experiment Reviewer: The user in Experimenter that is not the owner that has permission to review the experiment in both Experimenter and Remote Settings
+- Experimenter UI: The frontend application surface that Experimenter users interact with
+- Experimenter Backend: The backend server that hosts the Experimenter API and database
+- Experimenter Worker: The celery worker that can interact with the Experimenter database and make API calls to Remote Settings
+- Remote Settings UI: The frontend application surface that Remote Settings users interact with and is only accessible via whitelisted VPN access
+- Remote Settings Backend: The backend server that hosts the Remote Settings API and database and is only accessible via whitelisted VPN access
+
+### Preview
+
+A draft experiment that has been validly completed is marked for Preview, is published to the preview collection in Remote Settings, and is then accessible to specially configured clients.
+![](diagrams/preview.png)
+
+### Publish (Approve/Approve)
+
+A draft experiment that has been validly completed is reviewed and approved in Experimenter, is reviewed and approved in Remote Settings, and is then accessible to clients.
+![](diagrams/publish_approve_approve.png)
+
+### Publish (Reject/------)
+
+A draft experiment that has been validly completed is rejected by a reviewer in Experimenter. A rejection reason is captured in Experimenter and is displayed to the experiment owner in Experimenter.
```

Comment:
Re: "and is displayed to the experiment owner in Experimenter"

Wait, do we currently do that?

Reply:
This is a reorganization of the existing flow - more of the review process happens locally within Experimenter before moving onto Remote Settings. There will be a rejection reason that can be entered from the Experimenter side, not just Remote Settings.